src/Microsoft.OpenApi.OData.Reader/OpenApiConvertSettings.cs

```diff
@@ -275,6 +275,12 @@ public string PathPrefix
         /// </summary>
         public bool UseSuccessStatusCodeRange { get; set; } = false;
 
+        /// <summary>
+        /// Gets/Sets a value indicating whether to show the version of the assembly used for generating 
+        /// Open API document
+        /// </summary>
+        public bool ShowAssemblyInfo { get; set; } = true;
```

Comment:
What do you think about include or add assembly information instead? Because it's not a UI setting, nothing else is going to "show"

Reply:
Yes, `IncludeAssemblyInfo` makes more sense.